### PR TITLE
Support cpu burst strategy

### LINKED ISSUE #52: [feature request] add cpu burst strategy
**What would you like to be added**:

**Why is this needed**:

### PR DESCRIPTION
**Pull request by stormgbs**
Signed-off-by: bushuang.gbs <bushuang.gbs@alibaba-inc.com>

### Ⅰ. Describe what this PR does

fixes #52

### CODE REVIEW

File: pkg/util/config.go [hunk 1]
```diff
@@ -220,3 +221,14 @@ func DefaultCPUBurstConfig() slov1alpha1.CPUBurstConfig {
 		CFSQuotaBurstPeriodSeconds: pointer.Int64Ptr(-1),
 	}
 }
+
+type NodeCPUBurstCfg struct {
```
Comment (zwzhang0107) [inline pkg/util/config.go:225]: duplicated with config/config.go

File: pkg/util/config.go [hunk 2]
```diff
@@ -220,3 +221,14 @@ func DefaultCPUBurstConfig() slov1alpha1.CPUBurstConfig {
...
+	// An empty label selector matches all objects while a nil label selector matches no objects.
+	NodeSelector *metav1.LabelSelector `json:"nodeSelector,omitempty"`
+	*slov1alpha1.CPUBurstStrategy
+}
+
+type CPUBurstCfg struct {
```
Comment (saintube) [inline pkg/util/config.go:231]: These definitions are redudant with `CPUBurstCfg`, `NodeCPUBurstCfg` in `pkg/slo-controller/config/config.go`.

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Pull request merged**